modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/form21a_controller.rb

```diff
@@ -22,14 +31,33 @@ def submit
 
       attr_reader :parsed_request_body
 
+      def schema
```

Comment:
I think we can remove this private method for the sake of the test and stub the underlying `VetsJsonSchema` (see comment in the test below).